KubeArmor/core/containerdHandler.go

```diff
@@ -318,7 +321,7 @@ func (dm *KubeArmorDaemon) UpdateContainerdContainer(ctx context.Context, contai
 			dm.RuntimeEnforcer.RegisterContainer(containerID, container.PidNS, container.MntNS)
 		}
 
-		dm.Logger.Printf("Detected a container (added/%s/pidns=%d/mntns=%d)", containerID[:12], container.PidNS, container.MntNS)
+		dm.Logger.Printf("Detected a container (added/%s/pidns=%d/mntns=%d)", container.ContainerName, container.PidNS, container.MntNS)
```

Comment:
```suggestion
		dm.Logger.Printf("Detected a container (added/%.12s/pidns=%d/mntns=%d)", containerID, container.PidNS, container.MntNS)
```

Notice the `%.12s` that will limit the string size to 12 and will not panic otherwise even if the length is shorter.